pyccel/codegen/printing/fcode.py

```diff
@@ -1310,10 +1312,34 @@ def _print_PythonImag(self, expr):
     #========================== List Methods ===============================#
 
     def _print_ListAppend(self, expr):
-        target = expr.list_obj
+        target = self._print(expr.list_obj)
         arg = self._print(expr.args[0])
         return f'call {target} % push_back({arg})\n'
 
+    def _print_ListPop(self, expr):
```

Comment:
Could you please also support `listpop` as a function argument? like with `print`.
It fails in `Fortran`.

Reply:
Fixed : 67d9555e3bd1108dd000ff7c03ae2296af6f82d7
Test added : 71ee66ba68ee8e33311253f641c0bf9f7337a061

Reply:
Fixed : 67d9555e3bd1108dd000ff7c03ae2296af6f82d7
Test added : 71ee66ba68ee8e33311253f641c0bf9f7337a061